tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -1099,10 +1062,10 @@ def test_orderby_percentile_with_many_fields_multiple_entities_with_paginator(se
                 0,
                 0,
                 0,
+                0,
                 3,
                 1,
-                0,
-            ],  # [0, 0, 0, 3, 0, 1]
+            ],
```

Comment:
In general when I've made these sorts of changes to tests I try not to change the expectations but if you're confident about the correctness of these expected values then it's no problem

Reply:
> Looks good to me.
> 
> Not that it should block this change, but IMO inheritance in test classes (e.g. `BaseMetricsLayerTestCase`) makes them harder to understand and fix. Might be a practice to avoid going forward

I agree with you, indeed we are planning to spend some time refactoring all of our tests following a function-based approach. The problem with inheritance (and especially multiple inheritance) is the complexity introduced by the multiple dependencies that are very transparent.

Reply:
I did change this test by checking is throughout, not just to make it pass (I forgot to remove the comment). The reason of why they are in different order is because of the timeseries time interval and because right now we use the "at the hour" time this is should be the correct result.

Reply:
Appreciate your comments and thanks for the review @onewland!